tests/bundle/DependencyInjection/Compiler/ValueObjectVisitorResolverPassTest.php

```diff
@@ -7,33 +7,33 @@
 
 namespace Ibexa\Tests\Bundle\Rest\DependencyInjection\Compiler;
 
-use Ibexa\Bundle\Rest\DependencyInjection\Compiler\ValueObjectVisitorPass;
-use Ibexa\Contracts\Rest\Output\ValueObjectVisitorDispatcher;
+use Ibexa\Bundle\Rest\DependencyInjection\Compiler\ValueObjectVisitorResolverPass;
+use Ibexa\Contracts\Rest\Output\ValueObjectVisitorResolver;
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Definition;
 use Symfony\Component\DependencyInjection\Reference;
 
-class ValueObjectVisitorPassTest extends TestCase
+final class ValueObjectVisitorResolverPassTest extends TestCase
```

Comment:
I would add `declare(strict_types=1);` to the classes whenever the code is modified.